### ISSUE TITLE
Failed to create the arrow files when loading the .fragment.tsv file which is provided to benchmark the doublet detection 

### ISSUE THREAD

**Issue by secretloading** [GreenleafLab/ArchR#987]
Dear whom:
   Hi! I'm looking for an scATAC-seq set which can be used for benchmarking doublet detection. I looked into the GEO assesion GSE162690, and found that the provided file is of great use. I attemped to load the fragment file according to the ArchR pipline, while an error always occurred when creating the Arrowfiles. However, I also tried other fragment files and all succeeded. 
Here I attached the log file.
[ArchR-createArrows-94eb4fd1e72-Date-2021-08-17_Time-22-21-17.log](https://github.com/GreenleafLab/ArchR/files/7004042/ArchR-createArrows-94eb4fd1e72-Date-2021-08-17_Time-22-21-17.log)
 the log file 
Thank you very much!

**Comment by rcorces** [comment]
I am not able to recapitulate this error. These fragment files make arrow files for me.

**Comment by secretloading** [comment]
Could you please attach the commmand of creating the Arrowfiles? I just used the default parameters. All other fragment files succeeded while only these two files failed  when adding tile matrix. Have you adjusted any parameters when using the functions? Thank you so much.

**Comment by rcorces** [comment]
First, I downloaded the files in terminal using `wget`. Then:
```
> library(ArchR)
> addArchRGenome("hg19")
Setting default genome to Hg19.
> addArchRThreads(threads = 32)
Setting default number of Parallel threads to 32.

> inputFiles <- list.files(pattern = ".fragments.tsv.gz$")
> inputFiles
[1] "GSE162690_CellLine_HighLoading.fragments.tsv.gz" "GSE162690_CellLine_LowLoading.fragments.tsv.gz" 
> inputFiles <- inputFiles[2]
> 
> 
> inputFiles
[1] "GSE162690_CellLine_LowLoading.fragments.tsv.gz"
> 
> names(inputFiles) <- c("low")
> 
> inputFiles
                                             low 
"GSE162690_CellLine_LowLoading.fragments.tsv.gz" 
> 
> 
> ArrowFiles <- createArrowFiles(
+   inputFiles = inputFiles,
+   sampleNames = names(inputFiles),
+   filterTSS = 4, #Dont set this too high because you can always increase later
+   filterFrags = 1000, 
+   addTileMat = TRUE,
+   addGeneScoreMat = TRUE,
+   force = TRUE
+ )
filterFrags is no longer a valid input. Please use minFrags! Setting filterFrags value to minFrags!
filterTSS is no longer a valid input. Please use minTSS! Setting filterTSS value to minTSS!
Using GeneAnnotation set by addArchRGenome(Hg19)!
Using GeneAnnotation set by addArchRGenome(Hg19)!
ArchR logging to : ArchRLogs/ArchR-createArrows-28522d20f82060-Date-2021-08-18_Time-14-40-30.log
If there is an issue, please report to github with logFile!
2021-08-18 14:40:30 : Batch Execution w/ safelapply!, 0 mins elapsed.
2021-08-18 14:40:30 : (low : 1 of 1) Arrow Exists! Overriding since not completed!, 0 mins elapsed.
2021-08-18 14:40:30 : (low : 1 of 1) Reading In Fragments from inputFiles (readMethod = tabix), 0 mins elapsed.
2021-08-18 14:40:30 : (low : 1 of 1) Tabix Bed To Temporary File, 0.001 mins elapsed.
2021-08-18 14:40:47 : (low : 1 of 1) Successful creation of Temporary File, 0.289 mins elapsed.
2021-08-18 14:40:47 : (low : 1 of 1) Creating ArrowFile From Temporary File, 0.289 mins elapsed.
2021-08-18 14:41:00 : (low : 1 of 1) Successful creation of Arrow File, 0.501 mins elapsed.
2021-08-18 14:41:15 : createArrowFiles has encountered an error, checking if any ArrowFiles completed..
ArchR logging successful to : ArchRLogs/ArchR-createArrows-28522d20f82060-Date-2021-08-18_Time-14-40-30.log
```

**Comment by secretloading** [comment]
Yes, as shown at the end of the output, createArrowFiles has encountered an error, checking if any ArrowFiles completed. Therefore, only tmp arrowfiles exist, and no complete arrowfiles were created.

**Comment by secretloading** [comment]
Also I have some more details to be confirmed with your help. Here, I present my pipeline and queations.
First, I loaded the two peak matrices ( GSE162690_CellLine_HighLoading-PeakMatrix-SE.rds.gz  and GSE162690_CellLine_LowLoading-PeakMatrix-SE.rds.gz ) in R. According to the statistics, I thought these two are the replicates as mentioned in the published paper. In the 2 peak matrices, the total number of cells is 27220, which is consistent with the doublet-filtered number in the paper. However, I tried to get the raw matrix which consists of 38072 cells.
  Therefore, I attempted to load the fragment file ( GSE162690_CellLine_HighLoading.fragments.tsv.gz and GSE162690_CellLine_LowLoading.fragments.tsv.gz ) according to the ArchR pipeline. According to the statistics in the aforementioned rds file, I believe the qc-passed standards are minTSS = 4 and minFrags = 1000. I inherited the parameter setting when loading the fragment files using CreateArrowFiles function. However, only two cells can pass the quality control based on such settings, which makes the creation of Arrowfile fail. I looked into the ArchR-log, and found that both Median TSS Enrichment scores of two sets are only about 1.3. 
![image](https://user-images.githubusercontent.com/38944614/130164049-d37f0068-37f2-4d79-a5e8-81bb01c5f788.png)
![image](https://user-images.githubusercontent.com/38944614/130164061-8f845a92-5152-409b-b65b-87cc12c2d070.png)

 So could you please help me set the parameters, which can yield the 38072 cells-embraced set. When I set the parameter minTSS=0 and minFrags=1000, all qc-passed cells are 33672, still fewer than 38072. By the way, when calling the function with addTileMat = TRUE, the process always crashed with an error showing that "ERROR Found in .addTileMat".
![image](https://user-images.githubusercontent.com/38944614/130164077-1091dc3e-9845-42bc-9cf6-60839464cb5f.png)

Could you please give me some advice on these problems? Thank you so much!

**Comment by secretloading** [comment]
> First, I downloaded the files in terminal using `wget`. Then:
> 
> ```
> > library(ArchR)
> > addArchRGenome("hg19")
> Setting default genome to Hg19.
> > addArchRThreads(threads = 32)
> Setting default number of Parallel threads to 32.
> 
> > inputFiles <- list.files(pattern = ".fragments.tsv.gz$")
> > inputFiles
> [1] "GSE162690_CellLine_HighLoading.fragments.tsv.gz" "GSE162690_CellLine_LowLoading.fragments.tsv.gz" 
> > inputFiles <- inputFiles[2]
> > 
> > 
> > inputFiles
> [1] "GSE162690_CellLine_LowLoading.fragments.tsv.gz"
> > 
> > names(inputFiles) <- c("low")
> > 
> > inputFiles
>                                              low 
> "GSE162690_CellLine_LowLoading.fragments.tsv.gz" 
> > 
> > 
> > ArrowFiles <- createArrowFiles(
> +   inputFiles = inputFiles,
> +   sampleNames = names(inputFiles),
> +   filterTSS = 4, #Dont set this too high because you can always increase later
> +   filterFrags = 1000, 
> +   addTileMat = TRUE,
> +   addGeneScoreMat = TRUE,
> +   force = TRUE
> + )
> filterFrags is no longer a valid input. Please use minFrags! Setting filterFrags value to minFrags!
> filterTSS is no longer a valid input. Please use minTSS! Setting filterTSS value to minTSS!
> Using GeneAnnotation set by addArchRGenome(Hg19)!
> Using GeneAnnotation set by addArchRGenome(Hg19)!
> ArchR logging to : ArchRLogs/ArchR-createArrows-28522d20f82060-Date-2021-08-18_Time-14-40-30.log
> If there is an issue, please report to github with logFile!
> 2021-08-18 14:40:30 : Batch Execution w/ safelapply!, 0 mins elapsed.
> 2021-08-18 14:40:30 : (low : 1 of 1) Arrow Exists! Overriding since not completed!, 0 mins elapsed.
> 2021-08-18 14:40:30 : (low : 1 of 1) Reading In Fragments from inputFiles (readMethod = tabix), 0 mins elapsed.
> 2021-08-18 14:40:30 : (low : 1 of 1) Tabix Bed To Temporary File, 0.001 mins elapsed.
> 2021-08-18 14:40:47 : (low : 1 of 1) Successful creation of Temporary File, 0.289 mins elapsed.
> 2021-08-18 14:40:47 : (low : 1 of 1) Creating ArrowFile From Temporary File, 0.289 mins elapsed.
> 2021-08-18 14:41:00 : (low : 1 of 1) Successful creation of Arrow File, 0.501 mins elapsed.
> 2021-08-18 14:41:15 : createArrowFiles has encountered an error, checking if any ArrowFiles completed..
> ArchR logging successful to : ArchRLogs/ArchR-createArrows-28522d20f82060-Date-2021-08-18_Time-14-40-30.log
> ```

Excuse me, do you still stay tuned on this issue?
